plugins/modules/smart_proxy.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: smart_proxy
+version_added: 1.4.0
+short_description: Manage Smart Proxies
+description:
+  - Create, update and delete Smart Proxies
+author:
+  - "James Stuart (@jstuart)"
+  - "Matthias M Dellweg (@mdellweg)"
+options:
+  name:
+    description:
+      - Name of the Smart Proxy
+    required: true
+    type: str
+  url:
+    description:
+      - URL of the Smart Proxy
+    required: true
+    type: str
+  download_policy:
+    description:
+      - The download policy for the Smart Proxy
+      - Only used for Katello installations.
```

Comment:
```suggestion
      - Only available for Katello installations.
```

(to match the other strings we have)

Reply:
I should not have taken it from hostgroup, where "used" is proper.